### ISSUE TITLE
openBinaryFile doesn't exist

### ISSUE THREAD

**Issue by Havetrouble** [OliverBalfour/obsidian-pandoc#121]
<img width="232" alt="pandoc" src="https://user-images.githubusercontent.com/104652256/166059696-b632139a-4902-4449-8e8f-284c803b5766.png">

Can anyone help me out? I've checked like a thousand times if my pandoc path and pdflatex path are correct, but it still apears that openBinaryFile doesn't exist.

**Comment by Studentenfutter** [comment]
I got the same error. When switching to Markdown Render mode in settings the file gets exported but with missing images.
![image](https://user-images.githubusercontent.com/11717270/168568614-0dc90ca8-ff14-4607-aedc-a310db98e7fc.png)

**Comment by Studentenfutter** [comment]
Upon a close look the cause seems to be that `pandoc` expects embedded images / asset files in the root folder of the vault, not in the folder of the article. If you copy the asset files to the roort directory of your vault pandoc will render your document correctly.

@OliverBalfour is this a known bug?

**Comment by kalmir** [comment]
I'm experiencing the same problem :(

**Comment by Studentenfutter** [comment]
Seems like Pull request #101 could resolve this - @OliverBalfour could you review / accept it?

**Comment by OliverBalfour** [comment]
@Studentenfutter yes, I will take a look at it in the next few weeks. Apologies for the belated reply, I've been overwhelmed with uni work :sweat_smile:

**Comment by mkazharov** [comment]
Yep this fix is much needed, as I keep my attachments in a separate folder in order to keep note list clean.

**Comment by hugofreire** [comment]
🙏

**Comment by yuriymosiyenko** [comment]
Same issue

**Comment by jonasbirkelof** [comment]
Same for me with version 0.4.1

**Comment by purierca** [comment]
Hi all and @OliverBalfour, just wanted to up this thread as I have the same issue because my attachments aren't in the root folder

**Comment by chrispearce1972** [comment]
Is that why im getting this error ??
![image](https://user-images.githubusercontent.com/36820335/214810623-91f1ce68-fa1b-42c7-b7a3-2d306b610d60.png)
I guess it is, as the image is in the "uploads" directory -

**Comment by fatsen** [comment]
v1.1.16 has the same problem. And after moving pictures to root folder mentioned by Studentenfutter, it's ok.

**Comment by Anber55** [comment]
I also have the same problem. I hope it can be fixed as soon as possible. It is impossible to put all thousands of pictures in the root directory. Thank you
